spec/integration/login_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'Log in page', type: :feature do
+  before(:each) do
+    @user = User.new(name: 'AdeJesu', email: 'ade@ade.com', password: 'AdeOye')
+
+    visit new_user_session_path
+    fill_in 'Email', with: 'ade@ade.com'
+    fill_in 'Password', with: 'AdeOye'
+    click_button 'Log in'
+    visit groups_path
+  end
+
+  describe 'integration' do
+    before(:each) { visit new_user_session_path(@user) }
+
+    scenario 'displays content' do
+      expect(page).to have_content 'Log in'
+    end
+
+    scenario 'Submit form with incorrect email and password' do
+      within 'form' do
+        fill_in 'Email', with: 'ade@ade.com'
+        fill_in 'Password', with: 'aaaaaaa'
+      end
+      click_button 'Log in'
+      expect(page).to have_content 'Forgot your password'
+    end
+
+    scenario 'displays email' do
+      expect(page).to have_content('Email')
+    end
+
```

Comment:
- [ ] This test is failing. Kindly resolve it.

  ![urip](https://user-images.githubusercontent.com/54848363/202750731-a3f2110d-b517-4144-ac55-148b5a7e389f.jpg)

Reply:
@DammyShittu and to the next reviewer, all the test were passed from my side; I wouldn't know why @DammyShittu got the error. 

![test](https://user-images.githubusercontent.com/103386183/202753576-c6952cf2-be67-413d-94de-0e349d36aa6d.png)